test/fw/ptl/lib/pbs_ifl_mock.py

```diff
@@ -94,6 +94,7 @@
 ATTR_stageout = 'stageout'
 ATTR_resvTag = 'reserve_Tag'
 ATTR_resv_start = 'reserve_start'
+ATTR_resv_start_revert = 'reserve_start_revert'
```

Comment:
Do you need one of these for reserve_duration_revert?

Reply:
That's a good catch. Now that the attribute is only MGR read-only I'm not sure I need an entry here.  I think I should remove start_revert as well.
@bhroam what do you think?

Reply:
You should still have the ATTR_* macros, but they should be in pbs_internal.h.  I guess then they should be here too.

Reply:
@vstumpf I've addressed your comment.